hoth_api.yaml

```diff
@@ -319,5 +319,351 @@ components:
+            - General IaaS, PaaS, and SaaS
+            - Internet of Things (IoT)
+            - Networking
+            - Security
+            - Training
+        name:
+          type: string
+          maxLength: 100
+    PeriodOfPerformance:
+      description: "Part of DOW, represents a single Period of Performance"
+      properties:
+        base_period:
+          $ref: "#/components/schemas/Period"
+        option_periods:
+          type: array
+          items:
+            $ref: "#/components/schemas/Period"
+          maxItems: 4000
+        pop_start_request:
+          type: string
+          enum:
+            - Unselected
+            - Yes
+            - No
+        requested_pop_start_date:
+          type: string
+          format: date
+          example: "2021-07-01"
+        time_frame:
+          type: string
+          enum:
+            - No later than
+            - No sooner than
+        recurring_requirement:
+          type: string
+          enum:
+            - Unselected
+            - Yes
+            - No
+    GFEOverview:
+      description: "Part of DOW, represents GFE Overview fields"
+      properties:
+        dpas_unit_id:
+          type: string
+          maxLength: 6
+        property_custodian_name:
+          type: string
+          maxLength: 40 # This is another instance of maxLength in SNOW being weird, the person might have a name longer than 40 characters
+        dpas_custodian_number: # DPAS Custodian Number
+          type: string
+          maxLength: 6
+        property_accountable:
+          type: string
+          enum:
+            - Unselected
+            - Yes
+            - No
+        gfe_or_gfp_furnished:
+          type: string
+          enum:
+            - Unselected
+            - Yes
+            - No
+    ContractConsiderations:
+      description: "Part of DOW, represents Contract Considerations Table"
+      properties:
+        packaging_shipping_none_apply:
+          type: boolean
+        packaging_shippping_other:
+          type: boolean
+        packaging_shipping_other_explanation:
+          type: string
+          maxLength: 1000
+        potential_conflict_of_interest:
+          type: string
+          enum:
+            - Unselected
+            - Yes
+            - No
+        conflict_of_interest_explanation:
+          type: string
+          maxLength: 1600
+        contractor_provided_transfer:
+          type: boolean
+        contractor_required_training:
+          type: string
+          enum:
+            - Unselected
+            - Yes
+            - No
+        required_training_services: # This field looks incomplete on the SNOW side
+          $ref: "#/components/schemas/TrainingCourses"
+    TrainingCourses:
+      description: "Part of DOW, part of schema for ContractConsiderations, includes training courses name and value"
+      type: object
+      properties:
+        name:
+          type: string
+        value:
+          type: string
```

Comment:
I think this object will have a slightly different structure which was decided by the UI for keeping order of the trainings required. Something like the below from my understanding. @jasonburkert is this how you understood it as well?
```javascript
const training_courses = {
   "1": "Intro training",
   "2": "Basic training",
   "3": "Intermediate training"
}
```

Might be better for us to deal with this as an `array` (converting before the request) with the `name` being the `index`, since there could be any number of trainings provided or none at all. Or leave it as a `string` and handle it how we see fit in AWS. Not sure which is the right approach, just suggestion alternatives.